clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/model/Job.java

```diff
@@ -137,6 +139,7 @@ public Job(
     gpu = grpcJob.getJobDescriptor().getContainer().getResources().getGpu();
     networkMbps = grpcJob.getJobDescriptor().getContainer().getResources().getNetworkMbps();
     disk = grpcJob.getJobDescriptor().getContainer().getResources().getDiskMB();
+    signedAddressAllocations = new ArrayList<>();
```

Comment:
nit: Just assign this at the property declaration.

Reply:
updated